src/cloud/getting-started/quick-start.md

```diff
@@ -0,0 +1,89 @@
+- **Cloud Provider**: The cloud provider where your cluster will be deployed.
+- **Region**: The region where your cluster will be deployed.
+- **Database Name**: The name of the database that you will store vector data in. 
+
+![](../images/create_cluster.png)
+
+## Cluster Info 
+
+After creating a cluster, you will be directed to the cluster info page. Here you can find the details of your cluster, such as the connection string, the status of the cluster, and resource usage of the cluster.
+
+![](../images/cluster_info.png)
+
+We provide two types of connection endpoints:
+- Super User Endpoint: This connection string is used to connect to the cluster as a superuser(postgres). You can use this connection string to create databases, users, and manage the cluster.
+- Vector User Endpoint: This connection string is used to connect to the cluster as a vector user. You can use this connection string to store and query vector data.
+
+![](../images/two_types_connections.png)
+
+## Connect to the cluster
+
+To connect to the cluster, you can use the provided connection strings. You can connect to the cluster using the `psql` command-line tool. For detailed instructions on how to connect to the cluster, see [Connect to PGVecto.rs Cloud with psql](../connect/connect-with-psql.md). After connecting to the cluster, we can execute the following SQL query `dx` to validate you have successfully installed the vectors extension.
+
+```shell
+$ psql 'postgres://vector_user:56Cu2dtBt8WI@test-prgg3conlto0iayt.modelz.tech:5432/vector?sslmode=require'
+psql (15.3, server 16.2 (Debian 16.2-1.pgdg110+2))
+WARNING: psql major version 15, server major version 16.
+         Some psql features might not work.
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
+Type "help" for help.
+
+vector=> \dx
+                                                 List of installed extensions
+  Name   | Version |   Schema   |                                         Description
+---------+---------+------------+----------------------------------------------------------------------------------------------
+ plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
+ vectors | 0.2.1   | vectors    | vectors: Vector database plugin for Postgres, written in Rust, specifically designed for LLM
+(2 rows)
+```
+
+## Create Index
+
+In quick start, we will create a simple index. If you want to create a more advanced index, see [Reference](../../reference/). We provide rich user cases, see [Use Cases](../../use-case/). 
+
+```sql
+
+vector=> CREATE TABLE test (id integer PRIMARY KEY, embedding vector(3) NOT NULL);
+CREATE TABLE
+vector=> INSERT INTO test SELECT i, ARRAY[random(), random(), random()]::real[] FROM generate_series(1, 100) i;
+INSERT 0 100
+vector=> CREATE INDEX ON test USING vectors (embedding vector_l2_ops) WITH (options = "[indexing.flat]");
```

Comment:
This index does nothing. We can use HNSW since it's the most widely used one.

Reply:
lgtm